test/opening_device_test.py

```diff
@@ -94,7 +94,7 @@ def test_awning_str(self) -> None:
 
     def test_eq(self) -> None:
         """Testing eq method with positive results."""
-        pyvlx = PyVLX()
+        pyvlx = MagicMock()
```

Comment:
In a silent moment, you need to explain me what MagicMock actually does. Read a lot of Stackoverflow things, but never really understood ... :)

Reply:
Finally I'm not really an expert, just started with this unittests. It was probably one of the major points to create a custom component instead of improving native velux component in HomeAssistant devs. The HA devs want tests for the UI configuration of an integration and I never wrote such things before. However as this is my target sooner or later, I started now to learn that topic. [Here](https://realpython.com/python-mock-library/) is a bit lecture.

So far a mock or magicmock object substitutes and imitates a real object.  So in this case I used  this kind of "fake" pyvlx object in order to avoid loading a real pyvlx (which caused this stackoverflow). If you provide also a spec for a Mock, it will even pass an isinstance check. 
`pyvlx = MagicMock(spec=PyVLX)`

Finally the test here just compares two Blind objects if they are the same. If both receive the same mocked pyvlx object the test should pass.

For unittests mock objects provides usefull functions, for example if an object or function was called during a test, how often it was called, what attributes has been provided etc. In combination with patch, you can even catch a function on a completely different object and check if if was called during your test, like my first trial [here](https://github.com/pawlizio/pyvlx/blob/4d1d0b7d19c2bae99e9c160435234f9cb7f3c035/test/opening_device_test.py#L13-L22), where I wrote a test for the set_position() function of OpeningDevice class.